packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -74,6 +74,7 @@ export interface CheckboxTokens extends FontTokens, IForegroundColorTokens, IBac
   /**
    * States that can be applied to a checkbox
    * These can be used to modify styles of the Checkbox when under the specified state.
+   * Note: 'hovered','focused','circular','labelIsBefore','large' are not supported for Android
```

Comment:
Please update similar comments in SPEC.md

Reply:
Updated